contrib/mac/framework/Makefile

```diff
@@ -143,7 +143,7 @@ endif
 	# cleanup unnecessary install outputs
 	rm $(DESTDIR)$(datarootdir)/julia/startup.jl
 	rm -rf $(DESTDIR)$(datarootdir)/icons $(DESTDIR)$(datarootdir)/applications $(DESTDIR)$(datarootdir)/appdata
-	find $(DESTDIR)$(prefix)/$(framework_directory) \( -name '.DS_Store' -o -name '.gitignore' -o -name Makefile -o -name .travis.yml -o -name .codecov.yml \) -delete
+	find $(DESTDIR)$(prefix)/$(framework_directory) \( -name '.DS_Store' -o -name '.gitignore' -o -name Makefile -o -name -o -name .codecov.yml \) -delete
```

Comment:
Excess command line flags

Reply:
Shouldn't the `-o -name`  in front of `.travis.yml` also be removed?

Reply:
That's what Keno was saying, yes. Fixed now.